libqtile/widget/memory.py

```diff
@@ -19,25 +19,31 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 from __future__ import division
+
+import psutil
+
 from libqtile.widget import base
 
 
 def get_meminfo():
+    mem = psutil.virtual_memory()
     val = {}
-    with open('/proc/meminfo') as file:
-        for line in file:
-            key, tail = line.split(':')
-            uv = tail.split()
-            val[key] = int(uv[0]) // 1000
-    val['MemUsed'] = val['MemTotal'] - val['MemFree']
+    val['MemUsed'] = int(mem.used / 1024 / 1024)
+    val['MemTotal'] = int(mem.total / 1024 / 1024)
+    val['MemFree'] = int(mem.free / 1024 / 1024)
```

Comment:
There are more parameters in psutil that are also in meminfo, we may want to copy more over to limit how much this is a breaking change.

Reply:
I'll pull over whatever I can find :)